### PR TITLE
#78 Don’t set default JavaFX static version

### PR DESCRIPTION
**Pull request by jperedadnr**

### CODE REVIEW

File: src/main/java/com/gluonhq/NativeBaseMojo.java [hunk 1]
```diff
@@ -134,7 +134,9 @@ private void configSubstrate() {
         if (javaStaticSdkVersion != null) {
             clientConfig.setJavaStaticSdkVersion(javaStaticSdkVersion);
         }
-        clientConfig.setJavafxStaticSdkVersion(javafxStaticSdkVersion);
+        if (javafxStaticSdkVersion != null) {
```
Comment (eugener) [inline src/main/java/com/gluonhq/NativeBaseMojo.java]: in both cases, check for `null` is redundant. Substrate uses default version if those are `null`

Comment (jperedadnr) [reply src/main/java/com/gluonhq/NativeBaseMojo.java]: It was fixed in my branch, with this commit https://github.com/jperedadnr/client-maven-plugin/commit/d3665660bc473f807a16c1eddc90af4d39f90ac5, but I don't know why it didn't make it to this open PR??

Comment (eugener) [reply src/main/java/com/gluonhq/NativeBaseMojo.java]: Strange... looks like Github is having issues? Try to close this PR and create a new one maybe?

Comment (jperedadnr) [reply src/main/java/com/gluonhq/NativeBaseMojo.java]: ok, I'll do that

**Comment by jperedadnr** [comment]
Missing commit, I'll do another PR.

**Pull request closed without merging**